packages/zosjobs/src/cli/submit/stdin/stdin.definition.ts

```diff
@@ -0,0 +1,45 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandDefinition, ICommandOptionDefinition } from "@zowe/imperative";
+
+export const StdinDefinition: ICommandDefinition = {
+    name: "stdin",
+    aliases: ["in"],
+    type: "command",
+    summary: "Submit a job read from standard in",
+    description: "Submit a job (JCL) passed to the command via the stdin stream. " +
+        "The command presents errors verbatim from the z/OSMF Jobs REST endpoints. " +
+        "For more information about z/OSMF Jobs API errors, see the z/OSMF Jobs API REST documentation.",
+    handler: __dirname + "/../Submit.shared.handler",
+    options: ([
+        {
+            name: "view-all-spool-content", aliases: ["vasc"],
+            description: "View all spool content for specified job ID",
```

Comment:
This description doesn't look quite right here?

Reply:
I've updated the description, originally it had been copied from the local-file definition but this suffers from the same issues, so have updated them to match the dataset command.